pkg/installation/patchversion_test.go

```diff
@@ -0,0 +1,24 @@
+package installation
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFindLatestPatch(t *testing.T) {
```

Comment:
I would rename all variables and functions containing the word `latest` because it is confusing, we are not getting the latest patch but the patch <= `cliVersion`.